modules/shared/src/main/scala/org/tessellation/schema/aci/StateChannelOutput.scala

```diff
@@ -0,0 +1,12 @@
+package org.tessellation.schema.aci
+
+import org.tessellation.schema.address.Address
+
+sealed trait StateChannelOutput {
+  val address: Address
+}
+
+object StateChannelOutput {
+  case class Snapshot(address: Address, bytes: Array[Byte]) extends StateChannelOutput
+  case class Other(address: Address) extends StateChannelOutput
```

Comment:
Does it make sens to even have this class if it doesn't bring almost any information with it about the actual output? 🤔

Reply:
You're right. I think I will use `Option[Output]`